spec/system/provider_interface/provider_applications_search_spec.rb

```diff
@@ -60,11 +61,40 @@
     and_the_relevant_tags_should_be_visible
   end
 
+  scenario 'can search applications by application number' do
```

Comment:
can we move this to it's own system spec? Although we have older examples of this we are attempting to move to one scenario per spec for simplicity and clarity, and we accept there might be duplication

Reply:
Happy to do this, the rationale for adding the spec in this file is that we're expecting to remove the feature flag, which would bring this down to a single scenario again.

Reply:
fair enough, let's keep as is